src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/FunctionsTests.cs

```diff
@@ -208,7 +208,13 @@ public void TestSignaturesV2_3_X()
             DataFrame df = _spark
                .Read()
                .Json($"{TestEnvironment.ResourceDirectory}people.json");
-            df = Broadcast(df);
+
+            Assert.IsType<DataFrame>(Broadcast(df));
+
+            Assert.IsType<DataFrame>(_spark.Range(10));
+            Assert.IsType<DataFrame>(_spark.Range(10, 100));
+            Assert.IsType<DataFrame>(_spark.Range(10, 100, 10));
+            Assert.IsType<DataFrame>(_spark.Range(10, 100, 10, 5));
 
             col = Coalesce();
```

Comment:
I actually meant if you can also replace these `col = ...` as well.

Reply:
like this?